test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -187,17 +187,24 @@ internal CommandRunnerResult RunDotnet(string workingDirectory, string args, boo
         }
 
         internal CommandRunnerResult PackProject(string workingDirectory, string projectName, string args, string nuspecOutputPath = "obj", bool validateSuccess = true)
-            => PackProjectOrSolution(workingDirectory, $"{projectName}.csproj", args, nuspecOutputPath, validateSuccess);
+        {
+            // We can't provide empty or spaces as arguments if we used `string.IsNullOrEmpty` or `string.IsNullOrWhiteSpace`.
+            if (nuspecOutputPath != null) args = $"{args} /p:NuspecOutputPath={nuspecOutputPath}";
```

Comment:
If you are gonna chnage it, use:

```suggestion
            if (nuspecOutputPath != null)
            {
                args = $"{args} /p:NuspecOutputPath={nuspecOutputPath}";
            }
```

See Rule 1 in https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#c-coding-style

Reply:
So sorry about that, it's a wrong patch.